python/strings_udf/strings_udf/tests/test_string_udfs.py

```diff
@@ -228,6 +231,13 @@ def func(st):
     run_udf_test(data, func, "bool")
 
 
+def test_string_udf_istitle(data):
+    def func(st):
+        return st.title()
```

Comment:
Did this pass tests? I expected the `title()` function should fail since we can't generate new strings.
```suggestion
        return st.istitle()
```